tests/core/admin.py

```diff
@@ -28,7 +33,7 @@ class Meta:
         name = "Export/Import only book names"
 
 
-class BookAdmin(ImportExportMixin, admin.ModelAdmin):
+class BookAdmin(ImportExportModelAdmin):
```

Comment:
Should this kinda fix be handled in a non-documentation PR?

Reply:
Ideally yes, but it is in test code and it is an equivalent change.  I put this in because the [docs](https://github.com/matthewhegarty/django-import-export/blob/documentation-updates/docs/advanced_usage.rst#admin-integration) refer to this and I wanted it to be correct.

Reply:
Makes sense!